src/directives/click_action.ts

```diff
@@ -49,6 +49,8 @@ export default [
                     return false;
                 }
 
+                e.preventDefault();
```

Comment:
Thanks! I think previously this was not required because `threema://...` is not actually a valid URL :slightly_smiling_face: